src/handlebars/getinvolved.handlebars

```diff
@@ -59,8 +59,8 @@
       <li><a href="https://github.com/AdoptOpenJDK/TSC" target="_blank">Technical Steering Committee (TSC)</a> -
           <a href="https://github.com/AdoptOpenJDK/TSC/wiki" target="_blank">(wiki)</a>
           - The starting point</li>
-      <li><a href="https://github.com/AdoptOpenJDK/support" target="_blank">support</a> -
-        <a href="https://github.com/AdoptOpenJDK/support/wiki" target="_blank">(wiki)</a>
+      <li><a href="https://github.com/AdoptOpenJDK/openjdk-support" target="_blank">support</a> -
+        <a href="https://github.com/AdoptOpenJDK/openjdk-support/wiki" target="_blank">(wiki)</a>
```

Comment:
perhaps remove the wiki link as we don't have one

Reply:
Done